recipes/anyrpc/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.8)
+
+project(test_anyrpc CXX)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+find_package(anyrpc REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} test_anyrpc.cpp)
+target_link_libraries(${PROJECT_NAME} PRIVATE anyrpc::anyrpc)
```

```suggestion
target_link_libraries(${PROJECT_NAME} PRIVATE anyrpc::anyrpc)
target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_11)
```
Prefer features based on cmake targets

---

ok done